dipy/reconst/tests/test_mcsd.py

```diff
@@ -56,7 +55,7 @@ def _expand(m, iso, coeff):
     return params
 
 
-@pytest.mark.skipif(not mcsd.have_cvxpy, reason="Requires CVXPY")
+@needs_cvxpy
```

Comment:
It seems you also need to update the decorator line 19 by adding the reason (`needs_cvxpy = pytest.mark.skipif(not have_cvxpy, reason="Requires CVXPY")`

see the error here: https://github.com/dipy/dipy/actions/runs/7094023985/job/19308524800?pr=2987#step:9:5832

Reply:
Done in 7ef0df1.